.github/workflows/fuzzer.yml

```diff
@@ -52,4 +52,25 @@ jobs:
     - uses: stefanzweifel/git-auto-commit-action@v4
       with:
         commit_message: changing report
-        file_pattern: '*/hfuzz_workspace/fuzzer/fuzz_json/HONGGFUZZ*  */hfuzz_workspace/fuzzer/fuzz_json/SIG*'
+        file_pattern: '*/hfuzz_workspace/fuzzer/fuzz_json/HONGGFUZZ*  */hfuzz_workspace/fuzzer/fuzz_json/*.fuzz'
+
+    # Checkout current and last commit for the diff
+    - name: Checkout commits  
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 2
+
+    # Check if any new crashes were added
+    - name: Check diff
+      id: check_file_changed
+      run:  echo "report=$(git diff --name-only HEAD^ HEAD -- test_fuzzer/hfuzz_workspace/test_fuzzer/*.fuzz)" >> $GITHUB_OUTPUT 
```

Comment:
change here test_fuzzer fir the name of the current workflow fuzzer (fuzz_json)

Reply:
Resolved: https://github.com/lambdaclass/cairo-vm/pull/1331/commits/e693103f603315b90fa0e70e3074e92602f164fd